tests/integration/targets/sysrc/main/main.yml

```
@@ -0,0 +1,309 @@
+---
+- name: Test on FreeBSD VMs
+  when:
+    - ansible_facts.virtualization_type != 'docker'
+    - ansible_facts.distribution == 'FreeBSD'
```

Right now the tests seem to be skipped: https://dev.azure.com/ansible/community.general/_build/results?buildId=3696&view=logs&jobId=4d531206-24f7-54b9-089e-262f7c61faec&j=45df77c6-caca-5fd7-5ee8-2044329051c3&t=a4c415b7-e907-5c3f-7805-828d0e280dbd This should work:
```suggestion
    - ansible_facts.os_family == 'FreeBSD'
```